tests/unit/test_users.py

```diff
@@ -67,6 +67,23 @@ def test_is_following(self):
         )
 
 
+class TestGhostUser(helper.UnitHelper):
+    """Test methods on Ghost User class."""
+
+    described_class = github3.users.User
+
+    def setUp(self):
+        """Use None to create a ghost user."""
+        self.session = self.create_session_mock()
```

Comment:
```suggestion
        super().setup()
```

There's also `after_setup()` for overriding things: https://github.com/sigmavirus24/github3.py/blob/08fe91c664b2631c151c53bd7b3cbba86c06faed/tests/unit/helper.py#L192-L194